src/components/Editor/FreeBusy/FreeBusy.vue

```diff
@@ -254,39 +380,100 @@ export default {
 			}
 		},
 	},
-	mounted() {
-	  // Move file picker into the right header menu
-		// TODO: make this a slot once fullcalendar support it
-		//       ref https://github.com/fullcalendar/fullcalendar-vue/issues/14
-		//       ref https://github.com/fullcalendar/fullcalendar-vue/issues/126
-		const picker = this.$refs.datePicker
-		// Remove from original position
-		picker.$el.parentNode.removeChild(picker.$el)
-		// Insert into calendar
-		this.$el.querySelector('.fc-toolbar-chunk:last-child').appendChild(picker.$el)
-	},
 	methods: {
+		handleSelect(arg) {
+			this.currentStart = arg.start
+			this.currentEnd = arg.end
+		},
+		save() {
+			this.$emit('update-dates', { start: this.currentStart, end: this.currentEnd })
+		},
+		addAttendee(attendee) {
+			this.$emit('add-attendee', attendee)
+		},
+		removeAttendee(attendee) {
+			this.$emit('remove-attendee', attendee)
+		},
 		loading(isLoading) {
 			this.loadingIndicator = isLoading
 		},
-		setCurrentDate(date, updatedViaCalendar) {
-		  this.currentDate = date
-			if (!updatedViaCalendar) {
-				const calendar = this.$refs.freeBusyFullCalendar.getApi()
+		handleActions(action, date = null) {
+			const calendar = this.$refs.freeBusyFullCalendar.getApi()
+			switch (action) {
+			case 'today':
+				calendar.today()
+				break
+			case 'left':
+				calendar.prev()
+				break
+			case 'right':
+				calendar.next()
+				break
+			case 'picker':
 				calendar.gotoDate(date)
+				break
 			}
+			this.currentDate = calendar.getDate()
 		},
 	},
 }
 </script>
 
 <style lang='scss' scoped>
+.icon-close {
+	display: block;
+	height: 100%;
+}
 .modal__content {
 	padding: 50px;
 	//when the calendar is open, it's cut at the bottom, adding a margin fixes it
 	margin-bottom: 95px;
+	&__actions{
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		margin-bottom: 20px;
+		&__select{
+			width: 260px;
+		}
+		&__date{
+			display: flex;
+			justify-content: space-between;
+			align-items: center;
+			& > *{
+				margin-left: 5px;
+			}
+		}
+	}
+	&__header{
+		h3{
+			font-weight: 500;
+		}
+		margin-bottom: 20px;
+		&__attendees{
+			&__user-bubble{
+				margin-right: 5px;
+			}
+		}
+	}
+	&__footer{
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		margin-top: 20px;
+		&__title{
+			h3{
+				font-weight: 500;
+			}
+			&__timezone{
+				color: var(--color-text-lighter);
+			}
+		}
+	}
+}
+:deep(.vs__search ) {
+	text-overflow: ellipsis;
 }
```

Comment:
is this a generic issue with NcSelect? then let's make sure this is tracked and fixed upstream

we can keep this as temporary fix

Reply:
https://github.com/nextcloud-libraries/nextcloud-vue/pull/5325